react/src/components/atoms/media/SiteLogo/index.js

```diff
@@ -37,7 +40,6 @@ SiteLogo.defaultProps = {
     width: 45,
     height: 45
   },
-  siteName: 'Mass.gov',
   title: 'Mass.gov homepage'
```

Comment:
```suggestion
  title: 'Mass.gov homepage'
```

this should also be removed from defaultProps